### ISSUE TITLE
Implementing driver segment (for Worker module - TurtleBot)

### ISSUE THREAD

**Issue by UmangBR** [amosproj/amos2022ss03-turtlebot-fleet-management#54]
## User story
1. As a SD
2. I want to implement the driver segment function
3. So that I can drive the robot along the line with the calculations from the other functions. And also check whether the robot is still on the line and react accordingly.

## Acceptance criteria
* Drive the robot along the line with the calculations from the other functions.
* Check whether the robot is still on the line and react accordingly.

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by Sebastian2023** [comment]
Still WIP, since the fleet management software is not yet able to generate valid orders. Moreover, driving quickly from A to B along a given path requires a more complex control algorithm than rotating the robot and driving straight at it.

 wiki.ros.org/navigation

**Comment by Sebastian2023** [comment]
The driver is now able to process a order by driving straight from node to node with rectangular turns in between. Demonstrated in #75.
